src/services/members/schemas.ts

```diff
@@ -75,18 +75,24 @@ const getMany = {
 };
 
 // schema for getting members by
-const getBy = {
+const getManyBy = {
   querystring: {
     type: 'object',
     properties: {
-      email: { type: 'string', format: 'email' },
+      email: {
+        type: 'array',
+        items: { type: 'string', format: 'email' },
+      },
+      additionalProperties: false,
     },
-    additionalProperties: false,
   },
   response: {
     200: {
       type: 'array',
-      items: { $ref: 'http://graasp.org/members/#/definitions/member' },
+      items: {
+        type: 'array',
```

Comment:
Initially the `single` call was returning an array of users. It is not set up currently, but the search could include other fields like name or creation date? So that's why it's an array. Likewise the `multiple` endpoint return an array of arrays of members given different set of search fields (here emails). I don't think we will allow multiple users for one email tho.

Reply:
Ok makes sense !